src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -71,7 +72,18 @@ export class DynamicFacetManager extends Component {
      */
     compareFacets: ComponentOptions.buildCustomOption<IDynamicFacetManagerCompareFacet>(() => {
       return null;
-    })
+    }),
+    /**
+     * Defines the maximum number of expanded facets inside the manager.
+     * The rest of the facets will be collapsed.
+     *
+     * **Note:**
+     * Will prioritize the facets that possess active values, then will prioritize the first facets.
+     * If more there are more facets that possess active values than the `maximumNumberOfExpandedFacets` option, it will override it.
```

Comment:
If the number of facets with active values exceeds the value of the `maximumNumberOfExpandedFacets` option, it overrides the option.

Reply:
Wow I made a nice typo 😅 That's why we need you guys!